docs/admin/README.md

```diff
@@ -15,7 +15,7 @@ each specific area.
 
 * **Installation** Howto's for VM and cloud native deployment
 
-* **Kubernetes Operation Guide** Day-to-day administration for cloud native
+* **Cloud Native Operation Guide** Day-to-day administration for cloud native
```

Comment:
I think this is something we didn't want to move from Gluu. This created a bit of a confusion to what we actually meant by Cloud Native. As far as our docs go this is purely Kubernetes and I think the naming **should not** change.